examples/tfengine/generated/team/cicd/configs/import.sh

```diff
@@ -23,7 +23,7 @@ do
   case $c in
     d) DIRS="${OPTARG}" ;;
     *)
-      echo "Invalid flag ${OPTARG}"
```

Comment:
Why not print both?

Reply:
It's already the option is illegal like below

```
$ ./import.sh -b
./import.sh: illegal option -- b
Usage: import.sh -d="dir1 dir2 ..."
``